docs/release-playbook.md

```diff
@@ -76,9 +76,11 @@ While the release is active, you should make sure to do the following:
     *   Before closing a release blocker issue, add a comment indicating how the issue was resolved, for better tracking (e.g. `cherry-picked in #XYZ` - see [this example](https://github.com/bazelbuild/bazel/issues/16629#issuecomment-1302743541))
 *   Periodically check downstream tests that are run against the release branch to catch any breakages early on in the process. If you see any failures that don't appear at HEAD, reach out to the Bazel team, open an issue if needed, and add the issue to the release blockers list.
 *   When enough PRs have been cherry-picked and the release is nearing a ready state, create a release candidate (see [below](#create-a-release-candidate)).
-    *   When a few days pass and no more release blockers show up, push the candidate as the release. Otherwise, rinse and repeat the steps above.
+    *   One week before pushing the release candidate, go through all the remaining issues on the milestone to triage and add the "nice-to-have" label.
```

Comment:
I think we only need to meet and triage one week before the *final* RC. Early on, there's no need to decide whether something is nice-to-have, if we still have a lot of time before the estimated actual release date.